redshift_sqlalchemy/dialect.py

```diff
@@ -38,8 +71,9 @@ class RedShiftDDLCompiler(PGDDLCompiler):
                       Column('id', Integer, primary_key=True),
```

Comment:
@jklukas @thisfred : You'll want to rebase on https://github.com/graingert/redshift_sqlalchemy/pull/18 after it's landed